core/src/main/java/com/threerings/getdown/util/LaunchUtil.java

```diff
@@ -75,22 +79,14 @@ public static boolean updateVersionAndRelaunch (
     }
 
     /**
-     * Reconstructs the path to the JVM used to launch this process.
-     */
-    public static String getJVMPath (File appdir)
-    {
-        return getJVMPath(appdir, false);
-    }
-
-    /**
-     * Reconstructs the path to the JVM used to launch this process.
-     *
+     * @param javaLocalDir JRE location within appdir
      * @param windebug if true we will use java.exe instead of javaw.exe on Windows.
+     * @return the path to the JVM binary used to launch this process.
      */
-    public static String getJVMPath (File appdir, boolean windebug)
+    public static String getJVMBinaryPath(File javaLocalDir, boolean windebug)
```

Comment:
renamed the method to avoid unintended behavior for library users. didn't leave previous one to avoid confusion between two methods